cmd/sealer/cmd/cluster/join.go

```diff
@@ -71,9 +93,14 @@ func NewJoinCmd() *cobra.Command {
 				return err
 			}
 
+			masters := strings.Split(newMasters, ",")
+			masterIpList := utilsnet.IPStrsToIPs(masters)
+			nodes := strings.Split(newMasters, ",")
+			nodeIpList := utilsnet.IPStrsToIPs(nodes)
```

Comment:
Here, because the following functions need to use the []net.Ip type, we need to convert it. Who has a better solution, or rewrite the functions? @kakaZhou719 @VinceCui @justadogistaken @allencloud